mpi-proxy-split/test/run_mana_mpi_regression_test.py

```diff
@@ -0,0 +1,202 @@
+import sys
```

Comment:
let's add `#!/usr/bin/env python3` here and add execute permission for the script.